pkg/systemd/services/service-details.jsx

```diff
@@ -701,8 +701,10 @@ const DeleteModal = ({ reason, name, handleCancel, handleDelete }) => {
                    <Button variant="link" isDisabled={inProgress} onClick={handleCancel}>{_("Cancel")}</Button>
                </>}
         >
-            {dialogError && <ModalError dialogError={_("Timer deletion failed")} dialogErrorDetail={dialogError} />}
-            {reason}
+            <Stack hasGutter>
+                {dialogError && <ModalError dialogError={_("Timer deletion failed")} dialogErrorDetail={dialogError} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221201-102811-6d301c18-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)